source/_components/sensor.hydroquebec.markdown

```diff
@@ -52,3 +59,15 @@ Configuration variables:
   - **yesterday_total_consumption**: Yesterday total consumption
   - **yesterday_lower_price_consumption**: Yesterday lower price consumption
   - **yesterday_higher_price_consumption**: Yesterday higher price consumption
+
+<p class='note'>
```

Comment:
There's no real reason for this to be in a note since it most of the text on the page.